survey-runner.tf

```diff
@@ -90,6 +90,7 @@ module "survey-runner-queue" {
   dns_zone_name = "${var.dns_zone_name}"
   aws_key_pair = "${var.aws_key_pair}"
   internet_gateway_id = "${module.survey-runner-vpc.internet_gateway_id}"
+  backup_retention_period = "${var.backup_retention_period}"
```

Comment:
the key `backup_retention_period` does not exist in `survey-runner-queue` this should be `ebs_snapshot_retention_days` so if change above is made this line could be 
`ebs_snapshot_retention_days = "${var. queue_ebs_snapshot_retention_days}"`